drivers/i2c/Kconfig

```diff
@@ -91,19 +91,20 @@ config I2C_NRF5_GPIO_SCL_PIN
 	  The GPIO pin to use for SCL.
 
 config I2C_STM32_V1
-	bool "STM32 V1 Driver (F1/L1/F4X)"
+	bool "STM32 V1 Driver (F1/F4X)"
 	depends on SOC_FAMILY_STM32
-	depends on SOC_SERIES_STM32F4X || SOC_SERIES_STM32F1X
+	depends on SOC_SERIES_STM32F1X || SOC_SERIES_STM32F4X
 	default n
 	help
-	  Enable I2C support on the STM32 F1, L1 and F4X family of processors
+	  Enable I2C support on the STM32 F1 and F4X family of processors
```

Comment:
I guess this modification is a merge issue. Please keep other series information